test/testcne/metrics_test.c

```diff
@@ -0,0 +1,101 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2022 Intel Corporation
+ */
+
+#include <stdio.h>           // for NULL, EOF
+#include <metrics.h>         // for metrics_register
+#include <tst_info.h>        // for tst_error, tst_end, tst_start, TST_FAILED
+
+#include "metrics_test.h"
+
+static int
+test_cb(metrics_client_t *client, const char *cmd __cne_unused, const char *params __cne_unused)
+{
+    cne_printf("Test Callback\n");
+    metrics_append(client, "\"name\":\"Metrics Test\"");
+    return 0;
+}
+
+static int
+metrics_test(void)
+{
+    const char *c;
+    int ret = 0;
+    metrics_client_t *client;
+    lport_stats_t stats = {0};
+    char lpname[]       = "test_lport";
+
+    client = calloc(1, sizeof(metrics_client_t));
```

Comment:
The conventional way of doing this is

```
client = calloc()
if (!client)
    return -1;

client->cmd = "/test"
```
This is a nit, but can you please change it?

Reply:
Done